components/clp-package-utils/clp_package_utils/scripts/archive_manager.py

```diff
@@ -158,6 +165,8 @@ def main(argv: typing.List[str]) -> int:
         logger.exception("Failed to load config.")
         return -1
 
+    dataset = validate_dataset(clp_config, parsed_args.dataset)
```

Comment:
technically, we can delay dataset validation in native/archive_manager.py?

if dataset is None, we simply don't append it in the command.